### PR TITLE
Refine && test ci workflow

### PR DESCRIPTION
**Pull request by fortianyou**
1. upload summary after TorchBench
2. split cronjobs
3. delete auto rebase from cronjobs

### CODE REVIEW

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh [hunk 1]
```diff
@@ -32,8 +32,10 @@ ln -s $benchmark_repo_dir torchbenchmark
 
 # benchmark 
 config_file=blade_bench.yaml
+bench_target=partial
 if [ $1 ] && [ $1 == "full" ] ; then
```
Comment (qiuxiafei) [inline pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:36]: nit: just `"$1" == "full"` works i think.

File: scripts/ci/ossutil [hunk 2]
```diff
@@ -0,0 +1,2 @@
+wget -nv https://gosspublic.alicdn.com/ossutil/1.7.13/ossutil64 -O ossutil && chmod +x ossutil
```
Comment (qiuxiafei) [inline scripts/ci/ossutil:1]: just install ossutil in docker image?

**Review by qiuxiafei — APPROVED** [review]
LGTM

**Review by zzpmiracle — APPROVED** [review]
LGTM!

File: .github/workflows/torchbench.yml [hunk 3]
```diff
@@ -19,6 +19,26 @@ jobs:
...
+  TorchBenchPartial:
+    if: github.event.schedule == '30 15 * * *'
+    uses: ./.github/workflows/reusable.yml
+    with:
+      name: torch-offcial-benchmark
+      base_image: bladedisc/bladedisc:latest-runtime-torch1.12.0-cu113
+      device: gpu-a10
+      dockerfile: docker/cronjobs/Dockerfile.torch.bench
+      extra_run_args: --cap-add=SYS_ADMIN --gpus all
+      extra_envs: -e OSS_ENDPOINT=${{ secrets.DOCKER_PASSWORD }} -e OSS_AK=${{ secrets.OSS_AK }} -e OSS_SK=${{ secrets.OSS_SK }}
```
Comment (zzpmiracle) [inline .github/workflows/torchbench.yml:41]: `OSS_ENDPOINT=${{ secrets.DOCKER_PASSWORD }}`

wrong keys?

**Comment by fortianyou** [comment]
Mark related: https://github.com/alibaba/BladeDISC/issues/450

**Pull request merged**